examples/tfengine/iam_members.hcl

```diff
@@ -0,0 +1,90 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# {{$recipes := "../../templates/tfengine/recipes"}}
+
+data = {
+  parent_type      = "folder" # One of `organization` or `folder`.
+  parent_id        = "12345678"
+  billing_account  = "000-000-000"
```

Comment:
billing_account is probably not necessary either

Reply:
storage_location is probably not necessary either. removed.